.devcontainer/Dockerfile

```diff
@@ -83,7 +83,8 @@ RUN set -eux \
     && apt-get update \
     && apt-get install --yes --quiet ${PYTHON_PACKAGES}
 
-COPY --from=ghcr.io/astral-sh/uv:0.6 /uv /bin/uv
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
```

Comment:
Is there a reason you’re adding uvx here?

Reply:
Just followed the UV docs how to Install UV. As i am not used uv in the past and running the setup to Install the venv i decided to use the recommended install method to avoid errors.

Reply:
But there are no errors currently.

also, I’m not sure the latest change is appropriate either, it should be pinned just like the main project

Reply:
I have not pinned it, because dependabot istn't able to update those deps, so i decided to use latest here, because there already was a mismatch between the version here and in the production dockerfile.

I can revert that line and try again.
I think also it is possible to use the normal dockerfile and only add a additional layer for devcontainers i think, but that would bei a greater change for a seperate pr.

Reply:
reverted this now but installed same uv version as the normal image does